src/domain/processes/hooks/useProcessPage.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback } from "react"
+import { useDispatch } from "react-redux"
+import { useNavigate } from 'react-router-dom'
+
+/* TODO: Remover legacy code */
+// import { setSearchProcessIdDialog, setSearchProcessIdDialogData } from "pages/diagram/features/bpmnSlice"
```

Comment:
Esses comentários serão removidos. Os códigos comentados serão utilizados, mas precisa ter o domínio do diagrama criado. Assim que o dominio for criado, esses comentários serão removidos